File: src/main/java/br/com/munif/openprojects/vicentebase/service/PaisService.java
/* Arquivo gerado utilizando VICGERADOR por munif as 14/03/2018 06:35:02 */
 /* Para não gerar o arquivo novamente coloque na primeira linha um comentário com  VICIGNORE , pode ser essa mesmo */
package br.com.munif.openprojects.vicentebase.service;

import br.com.munif.framework.vicente.application.BaseService;
import br.com.munif.framework.vicente.application.VicRepository;
import br.com.munif.openprojects.vicentebase.domain.localizacao.Pais;
import org.springframework.stereotype.Service;

/**
 *
 * @author GeradorVicente
 */
@Service
public class PaisService extends BaseService<Pais> {

    public PaisService(VicRepository<Pais> repository) {
        super(repository);
    }

}

File: front/project/src/app/pais/pais.service.ts
/* Arquivo gerado utilizando VICGERADOR por anderson as 13/03/2018 15:19:34 */
/* Para não gerar o arquivo novamente coloque na primeira linha um comentário com  VICIGNORE , pode ser essa mesmo */
import { Injectable } from '@angular/core';
import { Http, Headers, Response } from '@angular/http';
import { SuperService} from '../vic-components/comum/super-service';

@Injectable()
export class PaisService extends SuperService{

  constructor(http:Http) {
    super('pais',http);
  }

}